dev-manuals/test-plans/minigames/regex-game-frontend/plans/u-regex-game-2-highscore-update.md

```diff
@@ -0,0 +1,37 @@
+# Highscore update test (`u.regex-game-2`)
+
+Version: V1.0, 07.08.2022 \
+Author: Leon Layer \
+Tester: -
+
+## Description
+
+Tests if the high score increases correctly when a higher score is reached.
+
+## Precondition
+
+The game is in a state where the maximum possible higscore has not yet been reached.
+
+## Postcondition
+
+by playing, the current higscore will be exceeded and the higscore will be updated in the menu. 
+
+## Typical procedure
+
+1. the tester plays the game but does not yet reach the maximum higscore. 
+2. the tester sees his previous higscore in the menu. 
+3. by trying again, he surpasses his previous best performance.
+4. This will update the higscore in the meu.
```

Comment:
```suggestion
1. the tester plays the game but does not yet reach the maximum highscore. 
2. the tester sees his previous highscore in the menu. 
3. by trying again, he surpasses his previous best performance.
4. This will update the highscore in the menu.
```

Reply:
```suggestion
1. Play the game without reaching the highscore 
2. Inspect your (previous) highscore in the menu 
3. Play again, now surpassing the highscore
4. The new highscore is displayed in the menu
```